Crypto101.org

```diff
@@ -2111,6 +2111,20 @@ a fixed-number of constant-time operations. The result is that every
 block is produced with exactly the same number of operations,
 regardless of what the key is.
 
+<<ARX>>
+
+Both stream ciphers are based on ARX ciphers (Add-Rotate-XOR). A good characteristic of ARX is the fact that it prevents timing attacks, since it runs in constant time. On top of that, it performs well on CPUs, because it doesn't need hardware support to function well.
+
+Here is an example ARX operation:
+
+#+BEGIN_LATEX
+\begin{equation}
+x = x ⊕ (y ⊞ z) <<< n
```

Comment:
Whoa Nelly! Unicode symbols? In my LaTeX? There's no way this is going to work, right?